indexer-app/src/cmr/indexer/config.clj

```diff
@@ -46,18 +58,26 @@
   from metadata db."
   {:default "cmr_deleted_collection_revision.exchange"})
 
+(defconfig deleted-granule-revision-exchange-name
+  "An exchange that will have messages passed to it whenever a granule revision is removed
+  from metadata db."
+  {:default "cmr_deleted_granule_revision.exchange"})
+
 (defn queue-config
   "Returns the rabbit mq configuration for the indexer application."
   []
   (assoc (rmq-conf/default-config)
          :queues [(index-queue-name)
                   (all-revisions-index-queue-name)
+                  (deleted-granules-index-queue-name)
                   (provider-queue-name)]
          :exchanges [(ingest-exchange-name)
                      (deleted-collection-revision-exchange-name)
+                     (deleted-granule-revision-exchange-name)
                      (provider-exchange-name)]
          :queues-to-exchanges
-         {(index-queue-name) [(ingest-exchange-name)]
+         {(index-queue-name) [(ingest-exchange-name)
+                              (deleted-granule-revision-exchange-name)]
```

Comment:
Changed my mind about removing the separate deleted-granules-queue. We should add a separate mapping between the deleted-granules-queue and the deleted-granule-revision-exchange.